arduino/security/rice-box.go

```diff
@@ -1,9 +1,10 @@
+// Code generated by rice embed-go; DO NOT EDIT.
 package security
 
 import (
 	"time"
 
-	"github.com/GeertJohan/go.rice/embedded"
+	"github.com/cmaglie/go.rice/embedded"
```

Comment:
@cmaglie did you edit this file manually or is it generated?

Reply:
generated